src/plugins/embeddable/public/lib/embeddables/i_embeddable.ts

```diff
@@ -28,6 +28,11 @@ export interface EmbeddableInput {
   id: string;
   lastReloadRequestTime?: number;
   hidePanelTitles?: boolean;
+
+  /**
+   * List of action IDs that this embeddable should not render.
+   */
+  disabledActions?: string[];
```

Comment:
Feel free to ignore,
Just curious if this disabled actions configuration have to be static or if we should support something like:
```
disabledActions?: () => string[] | string[];
```
Just fantasising as context menu options are usually dynamic and dependant on some "context".

Reply:
I was just discussing it yesterday with Stacey, and embeddable inputs right now need to be serializable. But, I'm sure we can work around this limitation in the future if need be.